src/routes/resources/+page.server.ts

```diff
@@ -2,28 +2,54 @@ import type { PageServerLoad } from './$types'
 import { parse } from 'yaml'
 import fs from 'fs'
 
-let resources_path: string = "data/resources.yml"
+interface Resource {
+  title: string,
+  description: string,
+  url: string,
+  tags: string[]
+}
+
+const parseResourcesFromYaml = (filePath: string) => {
+  const file = fs.readFileSync(filePath, 'utf8')
+  const resources: Resource[] = parse(file)
+  return resources;
+}
+
+const generateUniqueTags = (resources: Resource[]) => {
+  const tags: string[] = []
 
-// Reading in resources
-const file = fs.readFileSync(resources_path, 'utf8')
-const yml_data = parse(file) // TODO: Create interface for this, and then use it to validate edits during CI/CD
+  for (const resource of resources) {
+    tags.push(...resource.tags);
+  }
+
+  return [...new Set(tags)]
+}
 
-// Creating a list of unique tags
-let tags: string[] = [];
-let resource;
-for (resource of yml_data) {
-  tags.push(...resource.tags)
+const removeEmojisFromStr = (str: string) => {
+  return str.replace(/[\u1000-\uFFFF]+/g, '').trim();
 }
-tags = [...new Set(tags)]
 
-//Sorting resources by newest (assuming newest is at the bottom of the file )
-const yml_data_sort = [...yml_data].reverse();
+const sortAlphabeticallyIgnoringEmojis = (a: string, b: string) => {
+  const aWithoutEmojis = removeEmojisFromStr(a);
+  const bWithoutEmojis = removeEmojisFromStr(b);
+
+  return aWithoutEmojis.localeCompare(bWithoutEmojis);
+}
```

Comment:
This isn't the goal of the sorting. From #142:

> Alphabetic sorting of tags in resources filter (for easier grouping). *Tags with emojis should be at the top of the list.*

We would like to prioritise the emoji tags which are special tags (and also flag particularly high quality resources).

Reply:
I'll fix this today, no problem. 

Thanks.